bodhi/server/services/updates.py

```diff
@@ -428,7 +428,7 @@ def query_updates(request):
     # when rendering the html, so we add this here.
     if request.accept.accept_html():
         return_values.update(
-            types=reversed(list(bodhi.server.models.UpdateType.values())),
```

Comment:
why do we remove that here ?

Reply:
the reversed was causing some issues with the tests, and TBH, the order here for types doesnt really matter, so i removed it.